ansible/ansible-version.yml

```diff
@@ -0,0 +1,14 @@
+---
+- hosts: all
```

Comment:
`all` is used to match any `--limit` filter, `run_once: true` is used to use any single host from the set and `connection: local` to avoid actual ssh roundtrip.

If `hosts: localhost` is used instead then the version is not actually enforced if `localhost` is not part of play.